packages/clay-card/src/CardWithNavigation.tsx

```diff
@@ -74,41 +74,56 @@ export const ClayCardWithNavigation: React.FunctionComponent<IProps> = ({
 				</ClayCard.AspectRatio>
 			)}
 
-			<ClayCard.Body>
-				{!horizontal && (
-					<>
-						<ClayCard.Description displayType="title" truncate>
-							{title}
-						</ClayCard.Description>
-						<ClayCard.Description displayType="text" truncate>
-							{description}
-						</ClayCard.Description>
-					</>
-				)}
-
-				{horizontal && (
-					<ClayCard.Row>
-						<span className="autofit-col">
-							<ClaySticker inline>
-								<ClayIcon
-									spritemap={spritemap}
-									symbol={horizontalSymbol}
-								/>
-							</ClaySticker>
-						</span>
-						<span className="autofit-col autofit-col-expand">
-							<span className="autofit-section">
+			{(horizontal || title || description) && (
+				<ClayCard.Body>
+					{!horizontal && (
+						<>
+							{title && (
 								<ClayCard.Description
 									displayType="title"
 									truncate
 								>
 									{title}
 								</ClayCard.Description>
+							)}
+
+							{description && (
+								<ClayCard.Description
+									displayType="text"
+									truncate
+								>
+									{description}
+								</ClayCard.Description>
+							)}
+						</>
+					)}
+
+					{horizontal && (
+						<ClayCard.Row>
+							<span className="autofit-col">
+								<ClaySticker inline>
+									<ClayIcon
+										spritemap={spritemap}
+										symbol={horizontalSymbol}
+									/>
+								</ClaySticker>
 							</span>
-						</span>
-					</ClayCard.Row>
-				)}
-			</ClayCard.Body>
+							{title && (
+								<span className="autofit-col autofit-col-expand">
```

Comment:
This would be a good place to use `ContentCol` 😂

Reply:
Yeah I will go through and update our other components once a cut is released. I think if we update it before releasing @clayui/layout then CI might break for other PRs

Reply:
Sounds good! 👍